Inital Main Menu integration

zenvin-dev opened the pull request:
- Import UI sprites
- Set up main menu UI
- Add `UIManager`, `UIController` and `UIManager` classes

zenvin-dev committed:
Add Main Menu
Add UI Setup
Add Nunito Font & Font Asset
Add Button Sprites
Add Title & Studio Logo Sprites

zenvin-dev committed:
Add Main Menu Scripts:
Add UIManager.cs
Add UIController.cs
Add UIReference.cs

zenvin-dev committed:
Change UI referencing
Add MainMenuController.cs
Rename UIReference.cs -> UIIndex.cs
Rename UIController.TryGet<T> -> UIController.TryGetByID<T>

fouman75 reviewed (commented):
- As discussed, can we rename UIIndex to UIWidget?
- Consider removing the spaces before function declaration and function calls.

Everything looks good otherwise!

fouman75 commented on SpaceScrapper/Assets/_Game/_Scripts/Runtime/User Interface/UIIndex.cs:
```diff
@@ -0,0 +1,48 @@
+
+namespace SpaceScrapper.UserInterface
+{
+	public sealed class UIIndex : MonoBehaviour
```
Rename to UIWidget

zenvin-dev committed:
- Add `SettingsMenuController.cs`
- Add methods in `MainMenuController.cs`
- Rename `UIIndex` -> `UIWidget`
- Fix formatting regarding spaces following method invocations

fouman75 reviewed (approved):
Looks good!
